### DISCUSSION TITLE
32 bit support

### DISCUSSION THREAD

**Discussion by michalw2-alt** [hassio-addons/app-tailscale#292] [General]
Is there a chance to keep maintenance support on 32bit OS?
I'm still running on 32bit HAOS.

**Comment by lmagyar** [comment]
Shouldn't armv7 work? I'm using it on an old rPI2 and on some rPI3 (instead of the aarch64).

**Comment by frenck** [comment]
It is expected that Home Assistant will drop support for i386 and armhf as well in the upcoming months. 

There is no point in maintaining them. For a reference, the Node-RED addon had 80K downloads in a week, of which just 40 downloads have been for those platforms. 

Even just considering used processing power building these, make no sense. Therefore, I am dropping (or already have dropped) i386 and armhf from all community add-ons.


../Frenck
